packages/core/src/test-runner/CommandTestRunner.ts

```diff
@@ -2,14 +2,16 @@ import { exec } from 'child_process';
 import * as os from 'os';
 
 import { StrykerOptions } from '@stryker-mutator/api/core';
-import { RunResult, RunStatus, TestRunner, TestStatus } from '@stryker-mutator/api/test_runner';
+import { RunOptions, RunResult, RunStatus, TestRunner, TestStatus } from '@stryker-mutator/api/test_runner';
 import { errorToString } from '@stryker-mutator/util';
 
 import { kill } from '../utils/objectUtils';
 import Timer from '../utils/Timer';
 
+type CommandFunction = (runOptions: RunOptions) => string;
```

Comment:
Please rename to `CommandProvider`. I think that communicates clearer it's intent.

Reply:
Makes sense